ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5655,4 +5655,120 @@ RFVSC2X:
   - return_fan_run_status_2
   - return_fan_speed_percentage_command_2
   implements:
-  - OPERATIONAL
\ No newline at end of file
+  - OPERATIONAL
+
+EXTOR:
+  description: "shade control with extension monitoring based on outside illuminance"
+  is_abstract: true
+  uses:
+  - shade_extent_percentage_command
+  - outside_illuminance_sensor
+  - low_limit_illuminance_setpoint
+  - high_limit_illuminance_setpoint
+
+DPHC2:
+  description: "Two-pipe heating and cooling control for radiant panels"
+  is_abstract: true
+  opt_uses:
+  - supply_water_isolation_valve_status
+  - return_water_isolation_valve_status
+  - supply_water_temperature_setpoint
+  uses:
+  - supply_water_temperature_sensor
+  - return_water_temperature_sensor
+  - water_riser_mode
+  - cooling_output_percentage_command
+  - heating_output_percentage_command
+
+DPHC4:
+  description: "Four pipe heating and cooling controls for radiant panels"
+  is_abstract: true
+  opt_uses:
+  - supply_water_isolation_valve_status
+  - return_water_isolation_valve_status
+  - heating_supply_water_temperature_setpoint
+  - chilled_supply_water_temperature_setpoint
+  uses:
+  - heating_supply_water_temperature_sensor
+  - heating_return_water_temperature_sensor
+  - chilled_supply_water_temperature_sensor
+  - chilled_return_water_temperature_sensor
+  - cooling_output_percentage_command
```

Comment:
same question as above.

Reply:
In DE-FRA-NEM32 there aren't because the RP itself has no impact/influence to the change over group it belongs from a hydronic point of view. but I listed it as optional because this gives us the chance to use this abstract in other projects with a more sophisticated implementation.